src/sandstorm/grain.capnp

```diff
@@ -739,7 +752,17 @@ struct RoleDef {
   # "default", then such sharing actions will be treated as having an empty permissions set (the
   # user can open the grain, but the grain is told that the user has no permissions).
   #
+  # This field should *not* be used to specify a default role hint for sharing!  For that, use
+  # `defaultSelection` immediately below.
+  #
   # See also `ViewSharingLink.RoleAssignment.none`, below.
+
+  defaultSelection @6 :Bool = false;
```

Comment:
I'm happy that this field will now exist. Did you mean to actually use it in this pull request? I don't see any other uses of the the string "defaultSelection" in the diff.